ostd/src/arch/x86/mm/mod.rs

```diff
@@ -60,20 +60,27 @@ bitflags::bitflags! {
     }
 }
 
+/// Flush any TLB entry that contains the map of the given virtual address.
+///
+/// This flush performs regardless of the global-page bit. So it can flush both global
+/// and non-global entries.
 pub(crate) fn tlb_flush_addr(vaddr: Vaddr) {
     tlb::flush(VirtAddr::new(vaddr as u64));
 }
 
+/// Flush any TLB entry that intersects with the given address range.
 pub(crate) fn tlb_flush_addr_range(range: &Range<Vaddr>) {
     for vaddr in range.clone().step_by(PAGE_SIZE) {
```

Comment:
The call to `clone` is unnecessary, right?

Reply:
`clone` here is needed to create an iterator by `.step_by()`, which will consume the range. We pass the range by reference here. The linter and the compiler optimization have taken good care of it.